> Successfully installed `py-horovod~cuda controllers=gloo tensor_ops=gloo` on macOS 10.15.6 with Python 3.8.5 and Apple Clang 12.0.0. `horovodrun --check-build` passes as well.

hmm I get this on first build attempt...
```CMake Error at cmake/Utilities.cmake:56 (message):
  Cannot specify both HOROVOD_GPU_OPERATIONS and HOROVOD_GPU_ALLREDUCE
  options.  Try unsetting one of these variables and reinstalling.
Call Stack (most recent call first):
  CMakeLists.txt:91 (set_gpu_op)
```

--- a/var/spack/repos/builtin/packages/py-horovod/package.py
+++ b/var/spack/repos/builtin/packages/py-horovod/package.py
@@ -11,7 +11,7 @@ class PyHorovod(PythonPackage, CudaPackage):
     homepage = "https://github.com/horovod"
     git      = "https://github.com/horovod/horovod.git"
 
-    maintainers = ['adamjstewart']
+    maintainers = ['adamjstewart', 'aweits']
 
     version('master', branch='master', submodules=True)
     version('0.20.2', tag='v0.20.2', submodules=True)

--- a/var/spack/repos/builtin/packages/py-horovod/package.py
+++ b/var/spack/repos/builtin/packages/py-horovod/package.py
@@ -140,8 +140,8 @@ class PyHorovod(PythonPackage, CudaPackage):
 
         # Tensor Operations
         if 'tensor_ops=nccl' in self.spec:
-            env.set('HOROVOD_GPU_OPERATIONS', 'NCCL')
             env.set('HOROVOD_GPU_ALLREDUCE', 'NCCL')
+            env.set('HOROVOD_GPU_ALLGATHER', 'NCCL')
             env.set('HOROVOD_GPU_BROADCAST', 'NCCL')
 
             env.set('HOROVOD_NCCL_HOME', self.spec['nccl'].prefix)

--- a/var/spack/repos/builtin/packages/py-horovod/package.py
+++ b/var/spack/repos/builtin/packages/py-horovod/package.py
@@ -11,7 +11,7 @@ class PyHorovod(PythonPackage, CudaPackage):
     homepage = "https://github.com/horovod"
     git      = "https://github.com/horovod/horovod.git"
 
-    maintainers = ['adamjstewart', 'aweits']
+    maintainers = ['adamjstewart', 'aweits', 'tgaddair']
 
     version('master', branch='master', submodules=True)
     version('0.20.2', tag='v0.20.2', submodules=True)

